pages/linux/eyewitness.md

```diff
@@ -0,0 +1,24 @@
+# EyeWitness
+
+> Python based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.
+> More information: <https://www.christophertruncer.com/eyewitness-triage-tool>.
```

Comment:
```suggestion
> More information: <https://github.com/FortyNorthSecurity/EyeWitness>.
```
I think it is better to link to their GitHub page as their usage guides seems to be updated frequently.